docs/index.rst

```diff
@@ -125,31 +125,31 @@ The Ocean SDK includes the :ref:`dwave_cli` and the following packages:
    It also includes several useful composites—layers of pre- and post-processing—that
    can be used with DWaveSampler to handle :term:`minor-embedding`, optimize chain strength, etc.
 
-.. dropdown:: **minorminer** --- Minor-embeds graphs. \
-   :link-badge:`https://docs.ocean.dwavesys.com/en/stable/docs_minorminer/source/sdk_index.html, "docs", cls=badge-primary fr text-white` \
-   :link-badge:`https://github.com/dwavesystems/minorminer, "code", cls=badge-secondary fr text-white`
+.. dropdown:: `minorminer <https://docs.ocean.dwavesys.com/en/stable/docs_minorminer/source/sdk_index.html>`_---Minor-embeds graphs. 
```

Comment:
Use a ref here?

Reply:
We need to add an `.. _index_minorminer:` line in the package